doc/UserGuide.md

```diff
@@ -103,6 +103,11 @@ Format: `clear`
 Exits the program.<br>
 Format: `exit`  
 
+## Sorting the entries alphabetically by name : `sort`
+Sorts addressbook alphabetically <br>
```

Comment:
Follow the style of the other commands (use a full stop instead of a space before the `<br>`).